railgun/config/variants/enterprise/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
```

Comment:
same as 
railgun/config/variants/enterprise-postgres/kustomization.yaml

Are you sure about so many duplicates ?

Reply:
They are different:

```
$ diff railgun/config/variants/enterprise/kustomization.yaml railgun/config/variants/enterprise-postgres/kustomization.yaml  
2c2,3                      
< - ../../base/
---
> - ../postgres
> - enterprise-resources.yaml
```
The new 2.x kustomization.yamls mirror the 1.x kustomization.yamls. Each provides an OSS DB-less, OSS Postgres, Enterprise DB-less, and Enterprise Postgres set of manifests.